opencensus/stats/stats.py

```diff
@@ -12,24 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from datetime import datetime
+
+from opencensus.metrics.export.metric_producer import MetricProducer
```

Comment:
I kept this consistent with other imports, but we should decide on class- or module-level imports across the library.

Reply:
#478